### DISCUSSION TITLE
How to become a member of EddieHub in Github ?

### DISCUSSION THREAD

**Discussion by izam-mohammed** [EddieHubCommunity/support#6474] [Q&A]
I really want to become a member of EddieHub and currently, there has 2.1k members

**Comment by NaNshekhar04** [comment]
Hey @izam-mohammed 
You can follow the following Steps to get started with your Amazing journey at EddieHub.

Steps to get started :-

Join the EddieHub GitHub Org by raising an [issue](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will receive an invitation)
Join our EddieHub [Discord](http://discord.eddiehub.org/) community, where you can chat with the community and also ask any questions
Want to learn more about GitHub, watch Eddie's video here [Getting started with GitHub](https://www.youtube.com/watch?v=BI18N3ZC2Es)
Practice the Open Source workflow by adding your name to our [hacktoberfest-practice](https://github.com/EddieHubCommunity/hacktoberfest-practice) repo (instructions are in the repo and/or watching this [video](https://www.youtube.com/watch?v=8B_JWf7pG20) by [Eddie](http://github.com/eddiejaoude))
Get more GREEN squares by adding your profile to our [LinkFree](https://github.com/EddieHubCommunity/LinkFree) (Open Source alternative to LinkTree) for your custom EddieHub url
Make your GitHub profile stand out with this [free text/video course](http://eddiejaoude.io/course-github-profile-landing) by [Eddie Jaoude](http://github.com/eddiejaoude), which also has quiz and certificate
Have a look at our Good First Issue Finder [project](https://github.com/EddieHubCommunity/good-first-issue-finder)
Append your Discord/Twitter username with "EddieHub" (for example "Eddie Jaoude | EddieHub")

**Reply by izam-mohammed** [reply]
thanks for sharing these

**Comment by AVI-SHUKLA1110** [comment]
hello @izam-mohammed ,here is your answer-
To get started with EddieHub, follow these steps:

Request to join the EddieHub GitHub Organization by opening an issue
Join the EddieHub Discord community to connect with others and ask questions
Enhance your GitHub skills by watching Eddie's video "Getting started with GitHub"
Get hands-on experience with the Open Source workflow by adding your name to the hacktoberfest-practice repository
Boost your online presence with a custom EddieHub URL by adding your profile to the LinkFree (Open Source alternative to LinkTree)
Improve your GitHub profile with Eddie Jaoude's free text/video course, which includes a quiz and certificate.
Check out the Good First Issue Finder project.
Display your affiliation with EddieHub by appending "EddieHub" to your Discord/Twitter username (e.g. "Eddie Jaoude | EddieHub").

hope u like this:)
